enable/testing.py

```diff
@@ -256,7 +256,9 @@ def mouse_move(self, interactor, x, y, window=None,
                                         alt_down=alt_down,
                                         control_down=control_down,
                                         shift_down=shift_down)
-        window.get_pointer_position.return_value = (x, y)
+        if isinstance(window, _MockWindow):
```

Comment:
> The get_pointer_position function isn't used by most consumers of the mouse_move

This is True but it is not always clear when this is the case (e.g. using a subclass of a Tool). So the user will end up with a very weird error down the line.

Ideally we should be able to fake the mouse move in any Window (e.g maybe sent the mouse move event to the Qt widget) . However, for the moment, I would suggest to raise a warning if the get_pointer_position is not mocked and update the docstring to mention that for the mouse_move to be properly fakes the operation `get_pointer_position` needs to be a mock instance